src/cpu/core_dyn_x86/decoder.h

```diff
@@ -193,9 +197,9 @@ static INLINE void decode_increase_wmapmask(Bitu size) {
 		}
 	}
 	switch (size) {
-		case 1 : activecb->cache.wmapmask[mapidx]+=0x01; break;
-		case 2 : (*(Bit16u*)&activecb->cache.wmapmask[mapidx])+=0x0101; break;
-		case 4 : (*(Bit32u*)&activecb->cache.wmapmask[mapidx])+=0x01010101; break;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).